lib/msal-browser/src/event/EventHandler.ts

```diff
@@ -26,6 +28,10 @@ export class EventHandler {
      */
     addEventCallback(callback: EventCallbackFunction): string | null {
         if (typeof window !== "undefined") {
+            if (this.eventCallbacks.size === 0) {
```

Comment:
Is there somewhere else we can add the even listener? This feels kinda hacky and brittle.

Reply:
That still doesn't give us a way to remove it without checking the callback length. I'd also be open to just making this opt-in or out as sort of a compromise between the two options

Reply:
@jasonnutter Changed the approach here to be opt-in, putting the onus on the developer to enable/disable these events if and when they want them